virtual-desktop/src/app/window-manager/mvd-window-manager/shared/window-manager.service.ts

```diff
@@ -58,6 +59,7 @@ export class WindowManagerService implements MVDWindowManagement.WindowManagerSe
   private _lastScreenshotPluginId: string = '';  
   private _lastScreenshotWindowId: number = -1;
   public showPersonalizationPanel: boolean = false;
+  private autoSaveInterval : number = 5000;
```

Comment:
This is good for testing but probably too frequent otherwise. 30 seconds, 1 minute, or 5 minutes seems like better defaults.

Reply:
Changed this to 60000